compiler/src/compiler/cst.rs

```diff
@@ -630,7 +728,24 @@ impl TreeWithIds for Cst {
             CstKind::Identifier { .. } | CstKind::Symbol { .. } | CstKind::Int { .. } => {
                 (None, true)
             }
-            CstKind::Text { .. } | CstKind::TextPart(_) => (None, false),
+            CstKind::Text { parts, .. } => {
+                let res = parts.find_by_offset(offset);
+                if matches!(
+                    &res,
+                    Some(Cst {
+                        kind: CstKind::TextInterpolation { .. },
+                        ..
+                    })
+                ) {
```

Comment:
```suggestion
                if let Some(Cst {
                    kind: CstKind::TextInterpolation { .. },
                    ..
                }) = &res
                {
```

However, I don't think this filtering is even necessary. `TextPart` and quotes won't generate any findings that could be filtered out here